tournament relese

Marsel1112 opened the pull request:
Добавил енум в роут
В TournamentDao изменил метод для получения новых турниров и добавил метод для получениия списка с датой, типом, и колличеством участников в турнире Добавил TournamentsServlet
tournaments.jsp
tournaments.css

Marsel1112 committed:
Добавил ссылку в home.jsp;
Добавил енум в роут
В TournamentDao изменил метод для получения новых турниров и добавил метод для получениия списка с датой, типом, и колличеством участников в турнире
Добавил TournamentsServlet
tournaments.jsp
tournaments.css

Defenx commented on src/main/java/dao/TournamentDao.java:
```diff
@@ -50,10 +51,22 @@ public void create(Tournament tournament) {
+            throw new RuntimeException(he);
+        }
+    }
+    public List<Object[]> findTournamentsAndCountParticipants(Status status) {
```
а у нас же у объекта Tournament, есть List<TournamentParticipant> 
и если будет необходимо просто у листа можно взять .size()

По факту метод должен возвращать List<Tournament> и HQL запрос предельно простой (Все остальное достанет сам Hibernte)

Marsel1112 committed:
Удалил из TournamentDao метод findTournamentsAndCountParticipants

Переписал TournamentService TournamentsServlet под метод indTournamentsWithTheDesiredStatus

tournament.jsp научил работать с entity  Tournament

Defenx reviewed (commented):
![image](https://github.com/user-attachments/assets/d2c22c33-c962-4f2a-84cf-cfdbee3a3bd0)
SINGLE_PLAYER непонятно для обычных обывателей)

Marsel1112 committed:
Добавил метод в TournamentService для перевода типа турнира на русский, исользуя енум
исправил косяк в TournamentsServlet со множесмтвом пробелов (там у меня была строчка, для проверки возвращаемого значения, строчку убрал пробелы забыл)

Изменил название метода с findTournamentsWithTheDesiredStatus
на
findTournamentsWithStatus
